libportability-gfx/src/conv.rs

```diff
@@ -430,3 +431,95 @@ pub fn map_image_acces(access: VkAccessFlags) -> image::Access {
 
     mask
 }
+
+pub fn map_subpass_contents(contents: VkSubpassContents) -> command::SubpassContents {
+    match contents {
+        VkSubpassContents::VK_SUBPASS_CONTENTS_INLINE => command::SubpassContents::Inline,
+        VkSubpassContents::VK_SUBPASS_CONTENTS_SECONDARY_COMMAND_BUFFERS =>
+            command::SubpassContents::SecondaryBuffers,
+
+        _ => panic!("Unexpected subpass contents: {:?}", contents),
+    }
+}
+
+pub fn map_polygon_mode(mode: VkPolygonMode, line_width: f32) -> pso::PolygonMode {
+    match mode {
+        VkPolygonMode::VK_POLYGON_MODE_FILL => pso::PolygonMode::Fill,
+        VkPolygonMode::VK_POLYGON_MODE_LINE => pso::PolygonMode::Line(line_width),
+        VkPolygonMode::VK_POLYGON_MODE_POINT => pso::PolygonMode::Point,
+        _ => panic!("Unexpected polygon mode: {:?}", mode),
+    }
+}
+
+pub fn map_cull_face(cull: VkCullModeFlags) -> Option<pso::CullFace> {
+    use super::VkCullModeFlagBits::*;
+
+    if cull == VK_CULL_MODE_NONE as _ { None }
```

Comment:
let's have a `match` here

Reply:
It's not that easy as types mismatch iirc.

Reply:
You can do `match VkCullModeFlagBits::from_bits(cull) {..}`

Reply:
wait, we don't use `bitflags` :D 
well, I could `transmute` + truncate it?